roles/matrix-coturn/defaults/main.yml

```diff
@@ -1,10 +1,10 @@
 matrix_coturn_enabled: true
 
 matrix_coturn_container_image_self_build: false
-matrix_coturn_container_image_self_build_repo: "https://github.com/instrumentisto/coturn-docker-image.git"
+matrix_coturn_container_image_self_build_repo: "https://github.com/coturn/coturn/tree/master/docker/coturn/alpine.git"
 
 matrix_coturn_version: 4.5.2
-matrix_coturn_docker_image: "{{ matrix_coturn_docker_image_name_prefix }}instrumentisto/coturn:{{ matrix_coturn_version }}"
+matrix_coturn_docker_image: "{{ matrix_coturn_docker_image_name_prefix }}coturn/coturn:{{ matrix_coturn_version }}"
```

Comment:
@thedanbob @spantaleev despite specifying an `alpine` Dockerfile above in `matrix_coturn_container_image_self_build_repo`, you use `debian` image here. To use `alpine` Docker image the `-alpine` postfix is required now:
```yaml
matrix_coturn_docker_image: "{{ matrix_coturn_docker_image_name_prefix }}coturn/coturn:{{ matrix_coturn_version }}-alpine"
```

Reply:
Good point! Fixed in 62c0587b6aaed84c

Reply:
Good catch! I didn't notice that the alpine image wasn't the default for a given version tag. (Really should have, since I did notice the new image was much bigger than the old one.)

I've submitted #1019 to switch to the alpine image.